app-dicts/dictd-devils/metadata.xml

```diff
@@ -1,5 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-<!-- maintainer-needed -->
+<maintainer type="person">
+	<name>Samuel May</name>
+	<email>ag.eitilt@gmail.com</email>
+	<description>Primary ebuild maintainer</description>
```

Comment:
It's obvious, skip it.

Reply:
That was mostly a "I'm not responsible for the original data" on the ones where there's no `<upstream>` tag.  I'll admit that meaning's not obvious, and have removed it.